aws/resource_aws_codebuild_report_group_test.go

```diff
@@ -179,22 +266,16 @@ func testAccCheckAWSCodeBuildReportGroupDestroy(s *terraform.State) error {
 			continue
 		}
 
-		resp, err := conn.BatchGetReportGroups(&codebuild.BatchGetReportGroupsInput{
-			ReportGroupArns: aws.StringSlice([]string{rs.Primary.ID}),
-		})
+		resp, err := finder.ReportGroupByArn(conn, rs.Primary.ID)
 		if err != nil {
 			return err
 		}
 
-		if len(resp.ReportGroups) == 0 {
-			return nil
+		if resp != nil {
+			return fmt.Errorf("Found Report Group %s", rs.Primary.ID)
 		}
 
-		for _, reportGroup := range resp.ReportGroups {
-			if rs.Primary.ID == aws.StringValue(reportGroup.Arn) {
-				return fmt.Errorf("Found Report Groups %s", rs.Primary.ID)
-			}
-		}
+		return nil
```

Comment:
Always `continue` in `CheckDestroy` loop in case there are multiple resources in the configuration. 😄  Reference: https://github.com/hashicorp/terraform-provider-aws/issues/15847

```suggestion
```

Reply:
👍